### PR TITLE
Reverting Blob Buffs Made on a Refactor PR

### PR DESCRIPTION
**Pull request by SpaceDragon00**
## About The Pull Request

This PR fixes some of the balance changes made as part of #56306, as it is a refactor PR and the comments say to revert all the balance changes made as a part of the PR

## Why It's Good For The Game

Unapproved balance changes begone

## Changelog
:cl:
fix: Reverted the normal blob's max HP and initial HP values, which were accidentally modified (Old: 30 and 30, New: 25 and 21)
fix: Fixed an issue where certain strains with bonus core regeneration were regenerating faster than intended
code: Used a few unimplemented defines from the PR, like for the Strong and Resource blobs
code: Added an initial HP define that is set as a normal blob's integrity upon Initialize
/:cl:

### CODE REVIEW

**Comment by SpaceDragon00** [comment]
Er-- I hadn't expected the auto labels to consider the words I used in the title. For clarity, this is just a fix PR

**Comment by remove32** [comment]
fix indentation per [Tabs, not spaces](https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#tabs-not-spaces)

**Comment by GuillaumePrata** [comment]
Me bullying people that cry about Blob being OP.
Me digging code and finding out the Blob max HP was increased by 20% and the spawn HP by almost 50%.

The 20% HP alone is so fucking huge, a lot of Blob structures like EMP are meant to be 1 shot by their counter but now survive with 5 HP, effectively **doubling** the number of ammo/hits you need to kill it.

**Comment by MrStonedOne** [comment]
Please make the changelog entries make sense out of the context of this pr for viewers reading the in game changelog.

**Comment by optimumtact** [comment]
>the comments say to revert all the balance changes made as a part of the PR

who's comments

**Comment by ZephyrTFA** [comment]
> Used a few unimplemented defines from the PR, like for the Strong and Resource blobs
> Added an initial HP define that is set as a normal blob's integrity upon Initialize

should not be under the fix label. You aren't fixing anything

**Comment by SpaceDragon00** [comment]
> who's comments

Please atomize out your balance changes - LemonInTheDark

https://github.com/tgstation/tgstation/pull/56306#issuecomment-769535646

The Balance/Rebalance label was then removed on February 2nd

**Comment by SpaceDragon00** [comment]
> 
> 
> > Used a few unimplemented defines from the PR, like for the Strong and Resource blobs
> > Added an initial HP define that is set as a normal blob's integrity upon Initialize
> 
> should not be under the fix label. You aren't fixing anything

Changed those to the code improvement label in that case

File: code/modules/antagonists/blob/blobstrains/networked_fibers.dm [hunk 1]
```diff
@@ -9,7 +9,7 @@
 	color = "#4F4441"
 	complementary_color = "#414C4F"
 	reagent = /datum/reagent/blob/networked_fibers
-	core_regen_bonus = 5
+	core_regen_bonus = 3
```
Comment (Kylerace) [inline code/modules/antagonists/blob/blobstrains/networked_fibers.dm:12]: this was 5 before the linked pr was made

File: code/modules/antagonists/blob/blobstrains/regenerative_materia.dm [hunk 2]
```diff
@@ -7,7 +7,7 @@
 	complementary_color = "#AF7B8D"
 	message_living = ", and you feel <i>alive</i>"
 	reagent = /datum/reagent/blob/regenerative_materia
-	core_regen_bonus = 20
+	core_regen_bonus = 18
```
Comment (Kylerace) [inline code/modules/antagonists/blob/blobstrains/regenerative_materia.dm:10]: this was 20 before the linked pr was made, but it did change point_rate_bonus from 3 to 1 so change that back

Comment (Kylerace) [inline code/modules/antagonists/blob/blobstrains/regenerative_materia.dm:10]: this was 20 before the linked pr was made, but it also changed point_rate_bonus from 3 to 1

File: code/modules/antagonists/blob/blobstrains/networked_fibers.dm (hunk 1 above)
Comment (Kylerace) [inline code/modules/antagonists/blob/blobstrains/networked_fibers.dm:12]: this was 5 before the linked pr was made

File: code/modules/antagonists/blob/blobstrains/regenerative_materia.dm (hunk 2 above)
Comment (SpaceDragon00) [reply code/modules/antagonists/blob/blobstrains/regenerative_materia.dm:10]: A blob regenerates using their base regen (2) added to their core regen bonus. So it was 20 before the PR, 22 after, and this brings it back down to 22. The point regen bonus is the same story, but they accounted for that one

Comment (SpaceDragon00) [reply code/modules/antagonists/blob/blobstrains/regenerative_materia.dm:10]: image
For reference

**Pull request merged**